tests/foreman/longrun/test_oscap.py

```diff
@@ -300,6 +318,134 @@ def test_positive_push_updated_content(self):
+        }
+        vm_values = {
+            'distro': DISTRO_RHEL7,
+            'hgrp': hgrp7_name,
+            'rhel_repo': rhel7_repo,
+        }
+        tailoringfile_name = gen_string('alpha')
+        file_path = settings.oscap.tailoring_path
+        tailoring_path = get_data_file(file_path)
+        with Session(self) as session:
+            set_context(session, org=ANY_CONTEXT['org'])
+            # Creates oscap content for rhel7
+            session.nav.go_to_oscap_content()
+            self.oscapcontent.update(
+                rhel7_content,
+                content_org=self.config_env['org_name']
+            )
+            set_context(session, org=self.config_env['org_name'])
+            # Creates host_group for rhel7
+            make_hostgroup(
+                session,
+                content_source=self.config_env['sat6_hostname'],
+                name=hgrp7_name,
+                puppet_ca=self.config_env['sat6_hostname'],
+                puppet_master=self.config_env['sat6_hostname'],
+            )
+            make_oscap_tailoringfile(
+                session,
+                name=tailoringfile_name,
+                tailoring_path=tailoring_path,
+            )
+            self.assertIsNotNone(
+                self.oscaptailoringfile.search(tailoringfile_name))
+            # Creates oscap_policy for rhel7.
+            make_oscappolicy(
+                session,
+                content=policy_values.get('content'),
+                host_group=policy_values.get('hgrp'),
+                name=policy_values.get('policy'),
+                period=OSCAP_PERIOD['weekly'],
+                profile=OSCAP_PROFILE['common'],
+                period_value=OSCAP_WEEKDAY['friday'],
+                tailoring=tailoringfile_name,
+                tailoring_profile=OSCAP_PROFILE['tailoring_rhel7']
+            )
+            distro_os = vm_values.get('distro')
+            with VirtualMachine(distro=distro_os) as vm:
+                vm.install_katello_ca()
+                vm.register_contenthost(
+                    self.config_env['org_name'],
+                    self.config_env['ak_name'].get(distro_os)
+                )
+                self.assertTrue(vm.subscribed)
+                vm.configure_puppet(rhel7_repo)
+                self.hosts.update(
+                    name=vm._target_image,
+                    domain_name=vm.domain,
+                    parameters_list=[
+                        ['Host', 'Host Group', vm_values.get('hgrp')],
+                        ['Host', 'Lifecycle Environment',
+                            self.config_env['env_name']],
+                        ['Host', 'Content View', self.config_env['cv_name']],
+                        ['Host', 'Reset Puppet Environment', True],
+                        ['Host', 'Openscap Capsule',
+                            self.config_env['sat6_hostname']],
+                    ],
+                )
+                session.nav.go_to_hosts()
```

Comment:
line to remove as hosts.update_host_bulkactions navigate to entity by self.navigate_to_entity()